tools/eval/README.md

```diff
@@ -22,6 +22,7 @@ Supported datasets:
 - [ICDAR](#icdar2003)
 - [IIIT5K](#iiit5k)
 - [Mini Supervisely](#mini-supervisely)
+- [OTB-2015](#otb-2015)
```

Comment:
Lets make it back to OTB-100 and otb-100 then.

Reply:
Done.